chapters/dashboard/flow.md

```diff
@@ -0,0 +1,33 @@
+## Flow Design GUI
+
+##### Our GUI is a visual interface that makes composing flows as easy as browsing Facebook. Design, configure and deploy your flows right from the browser!
```

Comment:
Should we use a name of another company in here, probably better to stay away?

Reply:
Exchanged it with "social media"